composer.json

```diff
@@ -62,23 +42,64 @@
         "typo3/cms-scheduler": "dev-main as 12.0",
         "typo3/cms-seo": "dev-main as 12.0",
         "typo3/cms-setup": "dev-main as 12.0",
+        "typo3/cms-styleguide": "dev-main",
         "typo3/cms-sys-note": "dev-main as 12.0",
         "typo3/cms-t3editor": "dev-main as 12.0",
         "typo3/cms-tstemplate": "dev-main as 12.0",
         "typo3/cms-viewpage": "dev-main as 12.0",
         "typo3/cms-workspaces": "dev-main as 12.0",
-        "ttn/tea": "^3.0",
-        "t3docs/examples": "dev-main",
-        "t3docs/site-package": "dev-main",
-        "typo3-documentation-team/speeddemo": "^11.4",
-        "typo3/cms-styleguide": "dev-main"
+        "typo3/coding-standards": "^0.5.5"
+    },
+    "repositories": {
```

Comment:
The repositories can be removed, as tea is alread required in a stable version and blog_example can be required with "^12.0".

Reply:
can we change that in a follow up? Out of scope of this PR

Reply:
Yup, go ahead.